x/auth/src/client/grpc.rs

```diff
@@ -46,7 +54,9 @@ where
         let req = AuthNodeQueryRequest::Account(request.into_inner().try_into()?);
         let response = self.app.typed_query(req)?;
         let response: AuthNodeQueryResponse = response.try_into()?;
-        let AuthNodeQueryResponse::Account(response) = response;
+        let AuthNodeQueryResponse::Account(response) = response else {
+            return Err(Status::internal("invalid response enum variant"));
```

Comment:
I think it's better to be consistent and say "An internal error occurred while querying the application state."